webapp/src/app.tsx

```diff
@@ -4096,7 +4096,27 @@ function initPacketIO() {
                 id: 'n/a', // TODO
                 data
             }, "*")
+        },
+        (type, payload) => {
+            if (type == pxt.HF2.CUSTOM_EV_JACDAC) {
```

Comment:
if the message name is in pxt.appTarget.simulator.messageSimulators, we could send it over, so we don't hardcode jacdac.